README.md

```diff
@@ -1,45 +1,43 @@
-level-ws
-========
+# level-ws
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
+> A basic WriteStream implementation for [levelup](https://github.com/level/levelup)
 
-**A basic WriteStream implementation for [LevelUP](https://github.com/rvagg/node-levelup)**
-
-[![Build Status](https://secure.travis-ci.org/Level/level-ws.png)](http://travis-ci.org/Level/level-ws)
-[![Greenkeeper badge](https://badges.greenkeeper.io/Level/level-ws.svg)](https://greenkeeper.io/)
+[![level badge][level-badge]](https://github.com/level/awesome)
+[![npm](https://img.shields.io/npm/v/level-ws.svg)](https://www.npmjs.com/package/level-ws)
+![Node version](https://img.shields.io/node/v/level-ws.svg)
+[![Build Status](https://img.shields.io/travis/Level/level-ws.svg)](http://travis-ci.org/Level/level-ws)
+[![dependencies](https://david-dm.org/Level/level-ws.svg)](https://david-dm.org/level/level-ws)
+[![npm](https://img.shields.io/npm/dm/level-ws.svg)](https://www.npmjs.com/package/level-ws)
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 
-[![NPM](https://nodei.co/npm/level-ws.png?downloads)](https://nodei.co/npm/level-ws/)
-
-**level-ws** provides the most basic general-case WriteStream for LevelUP. It was extracted from the core LevelUP at version 0.18.0 but is bundled with [level](https://github.com/Level/level) and similar packages as it provides a general symmetry to the ReadStream in LevelUP.
-
-**level-ws** is not a high-performance WriteStream, if your benchmarking shows that your particular usage pattern and data types do not perform well with this WriteStream then you should try one of the alternative WriteStreams available for LevelUP that are optimised for different use-cases.
+`level-ws` provides the most basic general-case WriteStream for `levelup`. It was extracted from the core `levelup` at version 0.18.0.
 
-## Alternative WriteStream packages
-
-***TODO***
+`level-ws` is not a high-performance WriteStream. If your benchmarking shows that your particular usage pattern and data types do not perform well with this WriteStream then you should try one of the alternative WriteStreams available for `levelup` that are optimised for different use-cases.
 
 ## Usage
 
-To use **level-ws** you simply need to wrap a LevelUP instance and you get a `createWriteStream()` method on it.
-
 ```js
 var level = require('level')
-var levelws = require('level-ws')
-var db = level('/path/to/db')
+var WriteStream = require('level-ws')
 
-db = levelws(db)
-db.createWriteStream() // ...
+var db = level('/path/to/db')
+var ws = WriteStream(db) // ...
 ```
 
-### db.createWriteStream([options])
+## API
+
+### `ws = WriteStream(db[, options])`
+
+Creates a [Writable](http://nodejs.org/docs/latest/api/stream.html#stream_class_stream_writable_1) stream which operates in **objectMode**, accepting objects with `'key'` and `'value'` pairs on its `write()` method.
```

Comment:
We could link to a specific version of node.js docs (that matches our `readable-stream` version)

Reply:
=> #79